docs/pages/ktable.vue

```diff
@@ -58,9 +61,12 @@
       </DocsShow>
 
       <!-- Frontend Sorting Example-->
-      <h3>Table with Default Sorting</h3>
+      <h3>Table with Sorting</h3>
       <p>
-        The <code>KTable</code> can be used with default sorting functionality, allowing you to sort data on the client side without the need for server requests. There are 4 permissible data types - <code>string</code>,<code>number</code>,<code>date</code> and <code>undefined</code>. Columns declared with <code>undefined</code> data type are not sortable. This example demonstrates a table with default sorting enabled.
+        The <code>KTable</code> can be used with sorting functionality, allowing you to sort data on the client side without the need for server requests. There are 4 permissible data types - <code>string</code>,<code>number</code>,<code>date</code> and <code>undefined</code>. Columns declared with <code>undefined</code> data type are not sortable. This example demonstrates a table with sorting enabled. To allow client side sorting, set the <code>sortable</code> attribute to <code>true</code>.
+      </p>
+      <p>
+        You can also notice that clicking the same header multiple times toggles the sort direction cyclically in the order of <code>asc</code>, <code>desc</code> and <code>none</code>. The <code>none</code> state is the default state when the table is loaded by default.
```

Comment:
Maybe we could just leave out the second sentence here in favor of explaining it below in the already existing dedicated section for the default state since there is more options?

Plus a few optional tweaks for clarity.

Something like:

```suggestion
       Clicking the same header multiple times toggles the sort direction cyclically in the order of ascending, descending, and unsorted.
```